### PR TITLE
Add new controller with not implemented message

### PR DESCRIPTION
**Pull request by TayGov**
# Description

Add new controller with not implemented message

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

postman and unit test
# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

## For bcgov contributors:

SPDBCSC-171

### CODE REVIEW

File: src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/controller/GetNextSessionIdController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GetNextSessionIdController {
+    @Autowired
+    EcrcServicesImpl ecrcServices;
+
+    @CrossOrigin(origins = "/**")
+    @GetMapping("/getNextSessionId")
+    public ResponseEntity<String> getNextSessionId(@RequestParam(required=true) String orgId) {
```
Comment (shaunmillargov) [inline src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/controller/GetNextSessionIdController.java]: Can you change the input variable to 'orgTicketId' instead or 'orgId' please Taylor. This is the actual input name used in webMethods. Thx.

Comment (TayGov) [reply src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/controller/GetNextSessionIdController.java]: I added it to goAuthenticateUser as well.

Comment (shaunmillargov) [reply src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/controller/GetNextSessionIdController.java]: Should be changed there too just to be consistent. Thx.

**Comment by shaunmillargov** [comment]
@TayGov : Minor conflict to resolve before we can merge.

**Pull request merged**